cmd/cloudshell_open/scripts.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/fatih/color"
+)
+
+var cmdColor = color.New(color.FgHiBlue)
+
+func runScript(dir, command string, envs []string) error {
+	fmt.Println(infoPrefix + "Running command:")
+	cmdColor.Printf("\t%s\n", command)
+
+	cmd := exec.Command("/bin/bash", "-c", "set -euo pipefail; set -x; " + command)
+	cmd.Env = envs
+	cmd.Dir = dir
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	cmd.Stdin = os.Stdin
```

Comment:
Idea:

we can replace os.Stdout and os.Stderr with `color.New(color.FgGray)` and pass here (because, *color.Color is an `io.Writer`).

That way, all cmd outputs will be grayed out. Food for thought.

Reply:
if that doesn't work, something like this should:


```
type mywriter struct { out io.Writer }

func (m myWriter) Write(p []byte) (int, error) {
    return color.New(color.FgGray).Fprintf(m.out, p)
}

use os.Stdout = myWriter{os.Stderr}
```

Reply:
I like it, but I'm probably not understanding something with Go, but here is what I get:

```
Cannot use 'color.New(color.FgBlue)' (type *Color) as type io.Writer in assignment Type does not implement 'io.Writer' as some methods are missing: Write(p []byte) (n int, err error) 
```

Reply:
Yeah the 2nd example I gave above can solve it

Reply:
I've updated this.  Let me know how it looks.